tox.ini

```diff
@@ -42,6 +42,15 @@ commands = pip install "Django<1.6.99"
            pip install -r requirements-test.txt
            {envpython} test_app/manage.py test tidings
 
+[testenv:py27_dj17]
+basepython = python2.7
+commands = pip install "Django<1.7.99"
+           pip install .
+           {envpython} test_app/manage.py test tidings
```

Comment:
I'm pretty sure you don't need this line and the line above it. It causes this testenv to run the tests twice which I don't think is needed.